src/cwt.rs

```diff
@@ -0,0 +1,114 @@
+use serde::{Deserialize, Serialize};
+use serde_cbor::Value;
+use std::collections::BTreeMap;
+
+pub use crate::claim::{self, Claim, Label, NumericDate};
+
+/// CWT claims set.
+#[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq, Eq)]
+pub struct ClaimsSet(BTreeMap<Value, Value>);
+
+impl ClaimsSet {
+    pub fn insert_claim<T: Claim>(&mut self, claim: T) -> Option<Value> {
+        self.0.insert(T::label().into(), claim.into())
+    }
+
+    pub fn get_claim<T: Claim>(&self) -> Option<&Value> {
+        self.0.get(&T::label().into())
+    }
```

Comment:
I think it would be good if you could restore your previous functions that allow a library user to store and retrieve dynamic claims, i.e. claims that don't have an implementation of the `Claim` trait.